src/TwigComponent/src/Command/ComponentDebugCommand.php

```diff
@@ -0,0 +1,265 @@
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\TwigComponent\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Finder;
+use Symfony\UX\LiveComponent\Attribute\LiveAction;
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
+use Symfony\UX\TwigComponent\Attribute\PostMount;
+use Symfony\UX\TwigComponent\Attribute\PreMount;
+use Symfony\UX\TwigComponent\ComponentFactory;
+use Symfony\UX\TwigComponent\Twig\PropsNode;
+use Twig\Environment;
+
+#[AsCommand(name: 'debug:twig-component', description: 'Display current components and them usages for an application')]
+class ComponentDebugCommand extends Command
+{
+    public function __construct(private string $twigTemplatesPath, private ComponentFactory $componentFactory, private Environment $twigEnvironment, private iterable $components)
+    {
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setDefinition([
+                new InputArgument('name', InputArgument::OPTIONAL, 'A component name'),
+                new InputOption('dir', null, InputOption::VALUE_REQUIRED, 'Show all components with a specific directory in templates', 'components'),
+            ])
+            ->setHelp(<<<'EOF'
+                The <info>%command.name%</info> display current components and them usages for an application:
+
+                    <info>php %command.full_name%</info>
+
+                Find all components with a specific directory in templates by specifying the directory name with the <info>--dir</info> option:
+
+                    <info>php %command.full_name% --dir=bar/foo</info>
+
+                EOF
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+        $name = $input->getArgument('name');
+        $componentsDir = $input->getOption('dir');
+
+        if (null !== $name) {
+            try {
+                $metadata = $this->componentFactory->metadataFor($name);
+            } catch (\Exception $e) {
+                $io->error($e->getMessage());
+
+                return Command::FAILURE;
+            }
+
+            $class = $metadata->get('class');
+            $type = null;
+            $allProperties = [];
+
+            if ($class) {
+                $type = 'AsTwigComponent';
+
+                if ($metadata->get('live')) {
+                    $type = 'AsLiveComponent';
+                }
```

Comment:
What about making the column title just "Live?". And if you are live, we print an "X". If not live, leave that cell empty.